Source/CarthageKit/Project.swift

```diff
@@ -832,8 +832,6 @@ public final class Project { // swiftlint:disable:this type_body_length
 		return createVersionFileForCommitish(commitish, dependencyName: projectName, buildProducts: frameworkURLs, rootDirectoryURL: self.directoryURL)
 	}
 
-	private let gitOperationQueue = SerialProducerQueue(name: "org.carthage.Constants.Project.gitOperationQueue")
```

Comment:
Why was this queue removed?

Reply:
No longer need that queue since `checkoutDependency` operations are already serialized.